packages/analytics-v1.1/package.json

```diff
@@ -76,12 +76,14 @@
     "url": "https://github.com/rudderlabs/rudder-sdk-js.git",
     "directory": "packages/analytics-v1.1"
   },
-  "dependencies": {
-    "component-emitter": "2.0.0",
-    "@segment/localstorage-retry": "1.3.0"
-  },
+  "dependencies": {},
   "devDependencies": {
-    "@rudderstack/analytics-js-common": "*"
+    "@rudderstack/analytics-js-common": "*",
+    "component-emitter": "2.0.0",
+    "@segment/localstorage-retry": "1.3.0",
+    "@lukeed/uuid": "2.0.1",
+    "get-value": "3.0.1",
+    "ramda": "0.30.0"
```

Comment:
Haven't checked other packages but ramda, get-value, uuid, localstorage-retry these npm pkgs are dependencies, not devDependencies.

Reply:
You are correct. I did that intentionally.

When we publish the v1.1 and v3 SDK NPM packages, the corresponding package.json will also be part of them as it is. If we define these dependencies as "prod", the users will be forced to install them. 
However, our final SDK is a rolled-up JavaScript bundle that does not need any external dependencies.
So, to prevent unnecessary dependency installations and carry over any potential vulnerabilities for our users, I marked them as "dev". Our development, bundling, and testing will have no effect because of this.